webservices/tasks/legal_docs.py

```diff
@@ -9,6 +12,14 @@
 
 logger = logging.getLogger(__name__)
 
+DAILY_MODIFIED_STARTING_AO = """
```

Comment:
Can't we reuse the `RECENTLY_MODIFIED_STARTING_AO` query for this purpose as well?

Reply:
I thought 8 hrs going back is not sufficient.

Reply:
That makes sense